restapi/src/main/java/io/stargate/web/resources/v2/schemas/ColumnsResource.java

```diff
@@ -81,9 +82,11 @@ public Response getAll(
           List<ColumnDefinition> columnDefinitions =
               tableMetadata.columns().stream()
                   .map(
-                      (col) ->
-                          new ColumnDefinition(
-                              col.name(), col.type().name(), col.kind() == Column.Kind.Static))
+                      (col) -> {
+                        String type = col.type() == null ? null : col.type().cqlDefinition();
```

Comment:
It seems that this logic repeats in multiple places, maybe we can refactor it to `createNullSafeColumnDefintion()` or smth similar?

Reply:
We need a more holistic refactor since so much is shared between v1 and v2. I'll create a ticket.